Brightspace/insights-assessment-quality-dashboard
# US132667 Modified validation to be in sync with backend

AQ Settings Items: Parameterized the min and max for NUMBER input fields so that correlation will be [-1,1] and std deviation will be [0,1]. This will sync validations in front end and back end.

Testing
   - Verify correlation threshold field throws error if not in range [-1,1]
   - Verify Std Dev threshold field throws error if not in range [0,1]
   - Verify error messages

Security, UX, docs, perf, devops: N/A
FYI @Brightspace/draco-vis
